model/CardResponseV3.ts

```diff
@@ -18,7 +18,7 @@ export interface CardResponseV3 extends SummaryCardResponseV3 {
 export namespace CardResponseV3 {
 	export const type = SummaryCardResponseV3.type.extend<CardResponseV3>({
 		expires: isly.fromIs("YearMonth", YearMonth.is),
-		usage: isly.fromIs("CardUsage", CardUsage.is),
+		usage: isly.fromIs("CardUsage", PaymentCardUsage.is),
```

Comment:
maybe the one in quotes doesn't matter but it maybe should be the same as the type name